src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
- [ ] Well done designing your app so far. However, there are still some missing requirements to implement in order to match the given design on Zeplin. Please refer back to the given design. 🙏
It has to follow exactly the original in the following aspects:
1. Colors
2. Typography
3. Spacings (margin and padding)
![Screenshot](https://github.com/DjibrilM/book-store/assets/47625407/e7e625ee-7fa2-4042-a2ff-e315212e5b73)
Initially, please re-style the buttons, and add the circular progress that exists in the original design. 👍